insights/core/spec_factory.py

```diff
@@ -71,6 +70,30 @@ def stream(self):
     def _stream(self):
         raise NotImplementedError()
 
+    def _clean_content(self):
+        """
+        Clean (Obfuscate, Redact, and Filter) the Spec Content ONLY when doing
```

Comment:
This `Filter` in docstrings should also be removed according to the code changes.

Reply:
Done